packages/issue-receiver/src/main.ts

```diff
@@ -0,0 +1,48 @@
+import * as core from '@actions/core'
+import * as gh from '@actions/github'
+import * as Webhooks from '@octokit/webhooks'
+
+// TODO: Generalise this to work on any object
+interface Notification {
+    'list-ops': Exercise
+}
+
+interface Exercise {
+    commit_sha: string
+    commit_message: string
+    pull_request_html_url: string
+    new_cases: string[]
+}
```

Comment:
This obviously shouldn't contain a hardcoded list of exercise names. How can I define it in a way that any key is of type `Exercise`?

Reply:
Easiest is:

```typescript
Record<string, Exercise>
```